File: pgm/ABC_Java/ABC_209_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final int zero = 0;
        final int A = Integer.parseInt (arr[0]);
        final int B = Integer.parseInt (arr[1]);
        System.out.println (Math.max (zero, B - A + 1));
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_209_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.math.BigDecimal
// import java.math.RoundingMode
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer
// import java.util.TreeMap

import kotlin.math.*

const val space: String = " "
const val one: Int = 1
const val zero: Int = 0

fun main() {
    val (A: Int, B: Int) = readln().split(space).map { it.toInt() }
    println(max(zero, B - A + one))
    kotlin.system.exitProcess(0)
}